src/backend/common/DependencyModule.cpp

```diff
@@ -51,11 +96,11 @@ DependencyModule::DependencyModule(const char* plugin_file_name,
     // TODO(umar): Implement handling of non-standard paths
     UNUSED(paths);
     if (plugin_file_name) {
-        string filename = libName(plugin_file_name);
-        AF_TRACE("Attempting to load: {}", filename);
-        handle = loadLibrary(filename.c_str());
+        auto fileNames = libName(plugin_file_name, "");
```

Comment:
consider changing the name of the `libName` function because it is returning multiple file names.

Reply:
:+1: took care